server/src/test/java/org/opensearch/search/aggregations/AggregationProcessorTests.java

```diff
@@ -48,15 +48,27 @@ public void testPreProcessWithOnlyNonGlobalAggregators() throws Exception {
     }
 
     public void testPostProcessWithNonGlobalAggregatorsAndSingleSlice() throws Exception {
-        testPostProcessCommon(multipleNonGlobalAggs, 1, 0, 2);
+        testPostProcessCommon(multipleNonGlobalAggs, 1, 0, 2, false);
+    }
+
+    public void testPostProcessWithNonGlobalAggregatorsAndSingleSliceWithProfilers() throws Exception {
+        testPostProcessCommon(multipleNonGlobalAggs, 1, 0, 2, true);
```

Comment:
Can we update the Aggregation IT tests for both global and non-global aggregation and profile enabled to verify that indeed `*AggregationCollectorManager` related stats is present in the output of profile result ?